src/actions/recipes/sendRecipeComment.ts

```diff
@@ -0,0 +1,63 @@
+"use server";
+
+import { Recipe } from "@prisma/client";
+import { revalidatePath } from "next/cache";
+import { z } from "zod";
+
+import { db } from "@/db";
+import { currentSession } from "@/lib/currentSession";
+import { commentSchema } from "@/schemas/commentSchema";
+import type { FormState } from "@/types/formState";
+import { formDataToNestedObject } from "@/utils/formDataToNestedObject";
+
+export const sendRecipeComment = async (
```

Comment:
```suggestion
export const createRecipeComment = async (
```
i would say this better describes the action

Reply:
shit i also picked it up but forgot to change, thanks!